docs/source/guide/explanation/additional_features/tiling.rst

```diff
@@ -0,0 +1,114 @@
+
+        [>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 650/650, 17.2 task/s, elapsed: 38s, ETA:     0s
+        ==== merge: 7.326097726821899 sec ====
+
+
+
+
+Enable Tiling via OTX Training CLI 
+**********************************
+
+Currently tiling is supported for both detection and instance segmentation models.
+
+.. code-block:: 
+
+    +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+    |    TASK   |                 ID                |  NAME |                                 BASE PATH                                 |
+    +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+    | DETECTION |  Custom_Object_Detection_Gen3_SSD |  SSD  |  otx/algorithms/detection/configs/detection/mobilenetv2_ssd/template.yaml |
+    | DETECTION | Custom_Object_Detection_Gen3_ATSS |  ATSS | otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml |
+    | DETECTION |   Custom_Object_Detection_YOLOX   | YOLOX | otx/algorithms/detection/configs/detection/cspdarknet_yolox/template.yaml |
+    +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+
+    +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+    |          TASK         |                               ID                               |           NAME           |                                           BASE PATH                                           |
+    +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+    | INSTANCE_SEGMENTATION |    Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50     |    MaskRCNN-ResNet50     |     otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/template.yaml    |
+    | INSTANCE_SEGMENTATION | Custom_Counting_Instance_Segmentation_MaskRCNN_EfficientNetB2B | MaskRCNN-EfficientNetB2B | otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/template.yaml |
+    +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+
+To enable tiling in OTX training, set the tiling_parameters.enable_tiling parameter to 1. Here's an example of enabling tiling for the SSD model template:
+
+.. code-block::
+
+    otx train Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects params --tiling_parameters.enable_tiling 1
```

Comment:
I suggest combining this example and "Customizing Tiling Parameters" in one section. Also, I would rather remove the "Run Tiling on OpenVINO Exported Model" section since no difference between ordinary models and we already have tutorials on that.

Reply:
I personally don't have problems with "Run Tiling on OpenVINO Exported Model" section, it's a duplication, yes, but it's convenient then all related to Tiling info is in the same place

Reply:
Can you please add another option, where we build a workplace and then manually update the ``template.yaml`` to use tiling and update tiling parameters?

`otx build Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects params 

cd workspace-DETECTION

*update template*
train 
`

Reply:
I would add here an option with otx build as well to make calling other train function easier and concise, like that:

```
Here's an example of enabling tiling for the SSD model template using workspace:

otx build Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects params

cd workspace-DETECTION

otx train params --tiling_parameters.enable_tiling 1 ...

Also we can update ``tiling_parameters`` adding them in ``template.yaml`` like that:

hyper_parameters:
...
  tiling_parameters:
      enable_tiling:
          default_value: 1.0


```

Reply:
@GalyaZalesskaya  I added a new section: `Enable Tiling via OTX Build`